x/lockup/simulation/operations.go

```diff
@@ -20,6 +21,9 @@ func RandomMsgLockTokens(k keeper.Keeper, sim *simtypes.SimCtx, ctx sdk.Context)
 		return nil, err
 	}
 	lockCoin := sim.RandExponentialCoin(ctx, sender.Address)
+	if lockCoin.Amount.LTE(sdk.ZeroInt()) {
+		return &types.MsgLockTokens{}, fmt.Errorf("cannot lock coin that is zero or negative")
+	}
```

Comment:
Can we instead make the action based simulator run `msg.ValidateBasic()` and just return a no-op based off of that?